Src/AutoFixture/FreezeOnMatchCustomization.cs

```diff
@@ -29,6 +29,7 @@ public class FreezeOnMatchCustomization : ICustomization
         /// <exception cref="ArgumentNullException">
         /// <paramref name="targetType"/> is <see langword="null"/>.
         /// </exception>
+        [Obsolete("Please use FreezeOnMatchCustomization(object, IRequestSpecification) instead.")]
```

Comment:
For a user so far calling `FreezeOnMatchCustomization(t)`, we're now telling him or her to use an overload where a second `IRequestSpecification` argument is required, where before, none was required. What should that user supply as that second argument?